### ISSUE TITLE
File Organization?

### ISSUE THREAD

**Issue by jimfoltz** [SketchUp/sketchup-ply#1]
Should the file locations and folder hierarchy be similar to  the STL exrension?

**Comment by thomthom** [comment]
Yes, it would probably be a good idea to model the project closely after the STL project so things are consistent and familiar.

By the way, you have some stuff in a dev branch for STL - anything we can merge?

**Comment by jimfoltz** [comment]
> By the way, you have some stuff in a dev branch for STL - anything we can merge?

No, will do a pull request when ready.
